app/models/financial_transaction.rb

```diff
@@ -3,15 +3,20 @@
 class FinancialTransaction < ActiveRecord::Base
   belongs_to :ordergroup
   belongs_to :user
+  belongs_to :financial_transaction_type
 
   validates_presence_of :amount, :note, :user_id, :ordergroup_id
   validates_numericality_of :amount, greater_then: -100_000,
     less_than: 100_000
 
   localize_input_of :amount
 
+  after_initialize do
+    self.financial_transaction_type = FinancialTransactionType.default unless self.financial_transaction_type
```

Comment:
Moving this to a (private) method and using `after_initialize` with a symbol would make it just even a bit more flexible (allow a plugin to override it).